ios_nose2/test_cases.py

```diff
@@ -26,12 +26,3 @@ def setUp(self):
         self.config_files.set_config_properties_filenames('properties.cfg', 'ios-properties.cfg',
                                                           'local-ios-properties.cfg')
         super(IosTestCase, self).setUp()
-
-
-class IosHybridTestCase(test_cases.AppiumTestCase):
```

Comment:
I don't need this class because the new app used has a Web view section which can be used for hybrid tests.